src/main/scala/org/globalforestwatch/summarystats/firealerts/FireAlertsExport.scala

```diff
@@ -0,0 +1,146 @@
+package org.globalforestwatch.summarystats.firealerts
+
+import org.apache.spark.sql.{Column, DataFrame}
+import org.globalforestwatch.summarystats.SummaryExport
+
+
+object FireAlertsExport extends SummaryExport {
+
+  override protected def exportGadm(summaryDF: DataFrame,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method is longer than 50 lines](https://app.codacy.com/manual/thomas-maschler/gfw_forest_loss_geotrellis/pullRequest?prid=5008474)